<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::Rooms::EditorWidget</class>
 <widget class="QMainWindow" name="UnTech::GuiQt::Rooms::EditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QToolButton" name="resetAnimationButton">
        <property name="toolTip">
         <string>Reset Animation</string>
        </property>
        <property name="text">
         <string>Reset Animation</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/animation-reset.svg</normaloff>:/icons/animation-reset.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="playButton">
        <property name="toolTip">
         <string>Play Animation</string>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/animation-play.svg</normaloff>:/icons/animation-play.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nextTilesetFrameButton">
        <property name="toolTip">
         <string>Next Tileset Frame</string>
        </property>
        <property name="text">
         <string>Next Tileset Frame</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/animation-skip.svg</normaloff>:/icons/animation-skip.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nextPaletteFrameButton">
        <property name="toolTip">
         <string>Next Palette Frame</string>
        </property>
        <property name="text">
         <string>Next Palette Frame</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/animation-skip.svg</normaloff>:/icons/animation-skip.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolBar" name="editorToolBar">
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="floatable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="region"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="ZoomableGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>gui-qt/common/graphics/zoomablegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
